docs/workforce-management-framework/topics/con-security-introduction.adoc

```diff
@@ -0,0 +1,61 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} cloud server gives developers a choice one of the following authentication and authorisation providers
+link:http://www.keycloak.org/[Keycloak] or link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} is build around a security interface, which abstracts the developer away from any underlying
+authentication and authorisation libraries.
+Both implementations can be swapped at any time without breaking internal API of the {WFM-RC-NameShort} modules and reference applications.
```

Comment:
"...without breaking internal API..." > "...without breaking `the` internal API..."